### ISSUE TITLE
[Screen Reader- Adaptive Cards- Import- Sample cards] After selection of any drop-down controls talkback does not read selected for that value.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6995]
https://microsoft.visualstudio.com/OS/_workitems/edit/28336203

**Comment by carlos-zamora** [comment]
This bug was filed on the Android visualizer, not an adaptive card. That is an internal dev tool. Hence, closing.
